_translator-files/po/de/Running-a-Server.po

```diff
@@ -445,219 +483,223 @@ msgid ""
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:233
 msgid "### Server List Filename"
-msgstr ""
+msgstr "### Verzeichnis Server - Server Liste"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:235
 msgid "Leave this field empty unless you need to run your Server as a [Directory](#3-directory). When in use, this holds the list of registered Servers whilst restarting the Directory. This prevents the server list appearing \"empty\" until the Servers re-register."
-msgstr ""
+msgstr "Lass dieses Feld leer, es sei denn, du möchtest deinen Server als [Verzeichnis Server](#3-directory) betreiben. Wenn das Feld benutzt wird, wird die Liste der registrierten Server angezeigt, während das Verzeichnis neu gestartet wird. Dies verhindert, dass die Server Liste \"leer\" erscheint, bis sich die Server neu registrieren."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:237
 msgid "### Delay panning"
-msgstr ""
+msgstr "### Delay Panning - Stereo Effekt"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:239
 msgid "This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal \"volume\" panning."
-msgstr ""
+msgstr "Diese Option nutzt kleine Unterschiede in der Ankunftszeit des Schalls zwischen den beiden Ohren. Sie erzeugt einen Stereoeffekt, der dem natürlichen menschlichen Gehör ähnlich ist, im Vergleich zum normalen \"Lautstärke\"-Panning."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:241
 msgid "### Start Minimised"
-msgstr ""
+msgstr "### Start Minimiert"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:243
 #, no-wrap
 msgid "**Windows users** - If you want the Server to start automatically on system start, enable the corresponding check box.\n"
-msgstr ""
+msgstr "**Windows-Benutzer** - Wenn du möchtest dass der Server beim Systemstart automatisch gestartet wird, aktiviere das entsprechende Kontrollkästchen.\n"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:245
 msgid "### Server status icon"
-msgstr ""
+msgstr "### Server-Status-Symbol"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:247
 msgid "When the Server is running, the operating system will show an icon in the system tray or status area to represent whether the Server is active:"
-msgstr ""
+msgstr "Das Betriebssystem zeigt ein Symbol in der Taskleiste oder im Statusbereich an, um anzuzeigen, ob der Server aktiv ist:"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:249
 msgid "<figure><img src=\"{% include img/en-screenshots/server-inactive.inc %}\" style=\"float:left; margin-right:10px;\" loading=\"lazy\" alt=\"Image of the Jamulus Server icon\"></figure> The Server is empty"
-msgstr ""
+msgstr "<figure><img src=\"{% include img/en-screenshots/server-inactive.inc %}\" style=\"float:left; margin-right:10px;\" loading=\"lazy\" alt=\"Image of the Jamulus Server icon\"></figure> Der Server ist leer"
```

Comment:
Language code

Reply:
Updated /en- to /de-
Took a note to change in all .po files